### ISSUE TITLE
Update ODBC driver page on firebirdsql.org.

### ISSUE THREAD

**Issue by fdcastel** [FirebirdSQL/firebird-odbc-driver#232]
[This page](https://firebirdsql.org/en/odbc-driver/) is in need of a serious overhaul. 😅

**Comment by irodushka** [comment]
Yes, you are absolutely right) I don’t have access to this page right now, I’ll try to request it asap))

**Comment by fdcastel** [comment]
Can we close this one?
